library/general/src/modules/Linuxrc.rb

```diff
@@ -141,6 +141,10 @@ def text
       InstallInf("Textmode") == "1"
     end
 
+    def reboot_timeout
+      !InstallInf("Reboot_timeout").nil?
```

Comment:
if reboot_timeout is integer with value of timeout, then this code works strangely. I expect it will return number or default if not specified? So if default is 0, I expect code like
```
InstallInf("Reboot_timeout").to_i
```

Reply:
ok, so when checking another pr, you want to return nil if it is not specified. So code should look like:

```
return nil unless InstallInf("Reboot_timeout")

InstallInf("Reboot_timeout").to_i
```